manifests/o/oxygendioxide/Deepvocal-Hitsuboku-Kumi-CHN-dv1/1.1.0/oxygendioxide.Deepvocal-Hitsuboku-Kumi-CHN-dv1.installer.yaml

```
@@ -1,13 +1,13 @@
-# Created with YamlCreate.ps1 v2.1.0 $debug=AUSU.5-1-19041-1320
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.1.0.schema.json
-
-PackageIdentifier: oxygendioxide.Deepvocal-Hitsuboku-Kumi-CHN-dv1
-PackageVersion: 1.1.0
-MinimumOSVersion: 10.0.0.0
-InstallerType: inno
-Installers:
-- Architecture: x64
-  InstallerUrl: https://github.com/oxygen-dioxide/hitsuboku-kumi-chn-deepvocal/releases/download/v1.1.0/Hitsuboku-Kumi-Chinese-deepvocal-1.1.0-dvtb1.exe
-  InstallerSha256: BA16EF195A8C73E52864399B1C7E294614015907593E77C311CA88E29C6EAF09
-ManifestType: installer
-ManifestVersion: 1.1.0
+# Created with WinGet Automation using Komac v1.10.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: oxygendioxide.Deepvocal-Hitsuboku-Kumi-CHN-dv1
+PackageVersion: 1.1.0
+InstallerType: inno
+ReleaseDate: 2021-05-31
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/oxygen-dioxide/hitsuboku-kumi-chn-deepvocal/releases/download/v1.1.0/Hitsuboku-Kumi-Chinese-deepvocal-1.1.0-dvtb2.exe
+  InstallerSha256: 8BA38371D552717DC1C90115495AE57A1D54DBC04A4197068DB6A0BF0D1FDCFF

```

```suggestion
  InstallerUrl: https://github.com/oxygen-dioxide/hitsuboku-kumi-chn-deepvocal/releases/download/v1.1.0/Hitsuboku-Kumi-Chinese-deepvocal-1.1.0-dvtb1.exe
  InstallerSha256: BA16EF195A8C73E52864399B1C7E294614015907593E77C311CA88E29C6EAF09
```
This got swapped at some point, which did not happen in:
* https://github.com/microsoft/winget-pkgs/pull/115970